src/utils/mailup.ts

```diff
@@ -293,21 +293,20 @@ export function MailUpTransport(
 
       const email = errorOrEmail.value;
 
-      const errorOrResponse = await sendTransactionalMail(
-        options.creds,
-        email,
-        fetchAgent
-      ).run();
-
-      if (isRight(errorOrResponse)) {
-        // tslint:disable-next-line:no-null-keyword
-        return callback(null, {
-          ...errorOrResponse.value,
-          messageId: mail.data.messageId
-        });
-      } else {
-        return callback(errorOrResponse.value, undefined);
-      }
+      sendTransactionalMail(options.creds, email, fetchAgent)
```

Comment:
just note that `sendTransactionalMail` cannot throw any exception

Reply:
you never know :)